Src/Newtonsoft.Json/Serialization/JsonObjectContract.cs

```diff
@@ -85,8 +85,8 @@ public JsonPropertyCollection CreatorParameters
 
         /// <summary>
         /// Gets or sets the override constructor used to create the object.
-        /// This is set when a constructor is marked up using the
-        /// JsonConstructor attribute.
+        /// This is set when a constructor is marked up using
+        /// [<see cref="JsonConstructorAttribute"/>].
```

Comment:
Some Christmas tree decoration for Json.NET. :) Seriously, though, I wasn't quite sure how to rephrase the duplication of the word "attribute" after introducing the cref, so I figured it might be smart to have a semi-code example right inside the comment. But since that isn't immediately obvious it's probably a bad idea after all. I'll remove the brackets.